### DISCUSSION TITLE
上传大文件超时

### DISCUSSION THREAD

**Discussion by yann-y** [bfenetworks/bfe#900] [Q&A (问题和回答)] [answered]
postman可以成功上传，代理会报错
WriteRequestError: readfrom tcp 192.168.1.184:60360->192.168.1.184:8888: read tcp 192.168.1.184:80->192.168.1.26:53839: i/o timeout

net::ERR_CONNECTION_RESET

**Comment by mileszhang2016** [comment, marked as answer]
建议看一下：
https://github.com/baidu/bfe-book/blob/version1/design/timeout/timeout.md
